crates/flux-middle/src/fhir.rs

```diff
@@ -280,6 +290,22 @@ pub enum FluxItem<'fhir> {
     Func(SpecFunc<'fhir>),
 }
 
+#[derive(Debug)]
+pub struct ForeignItem<'fhir> {
+    pub ident: Ident,
+    pub kind: ForeignItemKind<'fhir>,
+    pub owner_id: MaybeExternId<OwnerId>,
+    pub span: Span,
+    pub vis_span: Span,
+}
+
+#[derive(Debug)]
+pub enum ForeignItemKind<'fhir> {
+    Fn(FnSig<'fhir>),
+    Static,
+    Type,
+}
```

Comment:
Let's only have `Fn` as a variant here so we don't have to care about the other cases. We should also add the `Generics` part of it such that we can return it in `OwnerNode::generics`

```suggestion
pub enum ForeignItemKind<'fhir> {
    Fn(FnSig<'fhir>, Generics<'fhir>),
}
```